CI-and-Infrastructure.md

```diff
@@ -0,0 +1,43 @@
+# CI Pipeline
+An [Azure instance](https://dev.azure.com/dotnet/CommunityToolkit) runs our CI pipeline.
+Our Azure pipelines configuration is found at the root of the project, [azure-pipelines.yml](https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/main/azure-pipelines.yml)
```

Comment:
```suggestion
Our Azure pipelines configuration is found at the root of the repository, e.g. [azure-pipelines.yml](https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/main/azure-pipelines.yml)
```

Generalizing a bit as we'll have the wiki copied to multiple repos. May investigate later a way for the clone task to customize links per repo, but that's non-trivial.